hudi-common/src/main/avro/HoodieRestoreMetadata.avsc

```diff
@@ -38,7 +38,6 @@
      /* overlaps with 'instantsToRollback' field. Adding this to track action type for all the instants being rolled back. */
      {
        "name": "restoreInstantInfo",
-       "default": null,
```

Comment:
could you help me understand why the default value was removed?

Reply:
The default value of null doesn't match top a field with type array. Instead of removing the default value of `null` i now changed it to an empty array `[]`

Reply:
Yes, empty has the correct element type.